scripts/deploy.backend.sh

```diff
@@ -14,18 +14,67 @@ case $ENV in
     ;;
 esac
 
+II_CANISTER_ID="$(dfx canister id internet_identity --network "$ENV")"
+POUH_ISSUER_CANISTER_ID="$(dfx canister id pouh_issuer --network "$ENV")"
+# URL used by II-issuer in the id_alias-verifiable credentials (hard-coded in II)
+# Represents more an ID than a URL
+II_VC_URL="https://identity.ic0.app"
```

Comment:
The contrary, that's invalid for local and ci development no?

Reply:
No, that's not correct. Maybe I'll rename the variable, but in the library it's using this name.

That's why I added a comment about, but this is more of an ID than a URL. The credential will have this url hardcoded and we need to compare it with this.

Reply:
Oh it's an ID. Yeah would be nice to rename if possible. Not necessary in this PR but, generally speaking.

Reply:
As discussed in [this comment](https://github.com/dfinity/oisy-wallet/pull/1773#discussion_r1679239909) we chose to keep the name according to the convention.